e2e/testsuite/codec.go

```diff
@@ -114,3 +116,32 @@ func UnmarshalMsgResponses(txResp sdk.TxResponse, msgs ...codec.ProtoMarshaler)
 
 	return nil
 }
+
+// MustProtoMarshalJSON provides an auxiliary function to return Proto3 JSON encoded
+// bytes of a message.
+// NOTE: Copied from https://github.com/cosmos/cosmos-sdk/blob/971c542453e0972ef1dfc5a80159ad5049c7211c/codec/json.go
+// and modified in order to allow `EmitDefaults` to be set to false for ics20 packet marshalling.
+// This allows for the introduction of the memo field to be backwards compatible.
+func MustProtoMarshalJSON(msg proto.Message) []byte {
```

Comment:
should we update this func's godoc? Looks like the same one copy pasta'd from transfer